examples/booking_app/docs/continuous_delivery.md

```diff
@@ -0,0 +1,137 @@
+## Fastlane
+
+### Distribution repository
+
+Create a new Github repository used for storing files and credentials for distribution of your app.
+Clone the repository and open the directory in the terminal.
+Using the [rx_bloc_cli][rx_bloc_cli_link] package, execute the following command:
+
+```
+rx_bloc_cli create_distribution .
+```
+
+The `create_distribution` command will populate the directory with the file structure required for things to work.
+The generated `README` file contains steps on where to download and how to name the required files.
+Within the `android` and `ios` directories, paste the previously downloaded files and rename them accordingly.
+
+Make sure that the `MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD` environment variable is present.
```

Comment:
As discussed the MOBILE_DISTRIBUTION_ENCRYPTION_PASSWORD env variable should be provided with a command prompt